Added a parsing step by comma for servers still sending back CSV for …

johnrey1 opened the pull request:
…multiple cookies #1

ran into same issue getting back a multi-cookie list where origin server separated by comma.

added a split on comma before the regex step

Commit:
Added a parsing step by comma for servers still sending back CSV for multiple cookies #1

rflechner reviewed (commented):
Thank for your contrib, could you please add unit tests for this class to avoid any regression ?
They are missing in this project 😄

johnrey1 committed:
Merge branch 'master' of github.com:rflechner/ScrapySharp

johnrey1 committed:
Added unit tests to cookie parser logic, added regex for comma separated parsing, PR #23

johnrey1 commented:
added a few unit tests. it's been a long time looking at the code.  I tried to find the host server I used that returned me CSV cookies. I couldnt find it.
It's not standard to use CSV, just semi-colon separated. But I have run across old web servers that do.
I changed the code a bit, and created a separate regex to use in the case where a ';' does not exist in the cookie string.

let me know if this works!
